hypervisor/arch/x86/guest/instr_emul.c

```diff
@@ -1596,7 +1596,7 @@ vie_calculate_gla(enum vm_cpu_mode cpu_mode, enum cpu_reg_name seg,
 	uint8_t glasize;
 	uint32_t type;
 
-	ASSERT(seg >= CPU_REG_ES && seg <= CPU_REG_GS,
+	ASSERT(seg >= CPU_REG_SEG_FIRST && seg <= CPU_REG_SEG_LAST,
```

Comment:
add () for left and right bool expression

Reply:
Ok, thanks!